lib/generators/smailer/migration_generator.rb

```diff
@@ -1,4 +1,5 @@
 require "rails/generators/active_record"
+require_relative '../../smailer'
```

Comment:
Also, here the `smailer` gem and folder should be in the load path already, so it should be possible to just `require 'smailer'`. Does that fix the problem for you?